packages/express-backend/playlist.js

```diff
@@ -18,13 +18,24 @@ const playlistSchema = new mongoose.Schema(
       type: String,
       trim: true,
     },
-
     songs: [
       {
         type: mongoose.Schema.Types.ObjectId,
         ref: "Song",
       },
     ],
```

Comment:
Here would be where to add the data structure

```
comments :[
{ 
    type : mongoose.Schema.types.ObjectId,
    ref : "Comments",
} ] 
```
This would be an array of comments attached to each playlist